src/TitleDetails/TitleDetails.js

```diff
@@ -74,6 +79,8 @@ const TitleDetails = ({
 
   const openModal = useCallback(
```

Comment:
what does `openModal`? It's hard to understand from the name

Reply:
`openAddPieceModal` is much better